src/subword.rs

```diff
@@ -463,6 +497,11 @@ where
 
     #[inline]
     fn next(&mut self) -> Option<Self::Item> {
+        if let Some(string) = self.string {
+            self.string = None;
```

Comment:
```suggestion
        if let Some(string) = self.string.take() {
```
Nitpicking here, so feel free to ignore

Reply:
👍 fixed